0.1.1

```diff
@@ -1,5 +1,59 @@
 {
-   "instructionList":[
+  "instructionList": [
+    {
+      "sObjectName": "Event",
+      "instructionTypeName": "command__Event Layout",
+      "description": "Add the Repeat field Event Layout",
+      "taskList": [
+        {
+          "jiraTicket": "ac-549a",
+          "description": "Add Type field to AC Task Layout",
+          "actionType": "Field",
+          "actionTypeName": "IsRecurrence2",
+          "location": {
+            "position": "below",
+            "reference": "OwnerId",
+            "section": "Calendar Event Details",
+            "column": 0,
+            "behavior": "Edit"
+          }
+        },
+        {
+          "jiraTicket": "ac-549b",
+          "description": "Add Type field to AC Task Layout",
+          "actionType": "Field",
+          "actionTypeName": "Attendees",
+          "location": {
+            "position": "below",
+            "reference": "IsRecurrence2",
+            "section": "Calendar Event Details",
+            "column": 0,
+            "behavior": "Edit"
+          }
+        }
```

Comment:
There is another PR involving these steps and adding them to the 1.2 file. Remove ac-549 tickets from this PR since it is for ac-20043